src/mito2/src/cache.rs

```diff
@@ -270,33 +270,42 @@ impl CacheManagerBuilder {
             Cache::builder()
                 .max_capacity(self.sst_meta_cache_size)
                 .weigher(meta_cache_weight)
-                .eviction_listener(|k, v, _cause| {
+                .eviction_listener(|k, v, cause| {
```

Comment:
Also check `cause.was_evicted()` : https://docs.rs/moka/latest/moka/notification/enum.RemovalCause.html#method.was_evicted

Reply:
`was_evicted` simply checks the cause's variants, I think it's not necessary to be included in the original metrics, Grafana must has some way to derive a new "was evicted" metric from it.